ui/component/or-map/src/index.ts

```diff
@@ -436,6 +437,12 @@ export class OrMap extends LitElement {
     @property({type: Boolean})
     public useZoomControl: boolean = true;
 
+    @property({type: Object})
+    public geoJson?: GeoJsonConfig;
+
+    @property({type: Boolean})
+    public showGeoJson: boolean = true;
```

Comment:
Why do we need this boolean property? Surely if you don't want to showGeoJson then just set geoJson to undefined/null

Reply:
The geoJson property acts as an override.
So if it is `undefined`, it will fallback to the geoJson of the `manager.displayRealm`.
`showGeoJson` can be used to hide it no matter what.

Furthermore, it is also a customizable option of the Map widget in the Insights page, where you can hide/show the GeoJSON layer.

Reply:
The `geoJson` property seems to only contain the geo JSON that should be currently loaded on the map so therefore whatever higher level component is setting this can also decide whether it should be set based on some other dashboard widget option can't it?

Even if we can justify having both properties on `or-map` I don't see why you also need both on `mapwidget` for the reason above.

Reply:
Example: User is in the smartcity realm.
The behavior of the properties will be the following;
- `geoJson: undefined`: GeoJSON of the SmartCity realm will be loaded.
- `geoJson: {}`: GeoJSON object specified here will be loaded instead.

If combined with showGeoJson:
- `geoJson: undefined; showGeoJson: false`: Don't load the SmartCity GeoJSON, and hide the layer.